server/websocket_events.py

```diff
@@ -2,6 +2,7 @@
 EVENT_SEND_ERROR = 'error'
 EVENT_SEND_YOUR_TURN = 'your_turn'
 EVENT_LIST_USERS = 'list_users'
+EVENT_END_GAME = 'end_game'
```

Comment:
Event should be 'game_over', it's copied to the 'event' key of the JSON sent to the bot